tests/tests_data/streaming/test_dataset.py

```diff
@@ -527,3 +530,199 @@ def test_s3_streaming_dataset():
     dataset = StreamingDataset(input_dir="s3://pl-flash-data/optimized_tiny_imagenet")
     assert dataset.input_dir.url == "s3://pl-flash-data/optimized_tiny_imagenet"
     assert dataset.input_dir.path is None
+
+
+@pytest.mark.skipif(sys.platform == "win32", reason="Not tested on windows and MacOs")
+def test_resumable_dataset_single_worker(tmpdir):
```

Comment:
I saw we are not testing the effect of `current_epoch` when resuming, maybe worth adding

Reply:
Done !